contracts/ERC20.sol

```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: mit
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract ExampleERC20 is ERC20, AccessControl {
```

Comment:
A suggestion, we should make the contract `Ownable` instead of `AccessControl` since we're only using one role. You then would use the `onlyOwner` modifier.

Reply:
Resolved